Services/ReplayFileService.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
How about putting this file in its own folder?

Reply:
Since this is the only file, I'll leave it here for now.